src/config.ts

```diff
@@ -36,6 +36,11 @@ You can use multiple options in conjunction, e.g. "garbo nobarf ascend"',
       help: "*EXPERIMENTAL* garbo will sacrifice some optimal behaviors to run quicker. Estimated and actual profits may be less accurate in this mode.",
       default: false,
     }),
+    workshed: Args.string({
+      setting: "",
+      help: "workshed that garbo should intelligently switch to after fully exploiting the benefits of your current workshed.",
+      default: "",
+    }),
```

Comment:
Make this `Args.custom`, have it convert from a string to an item _here_, that way `help` can explicitly list the aliases we support

Reply:
I sure hope this doesn't result in a circular import somewhere down the line...